ashlar/migrations/0014_auto_20150929_1551.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.contrib.gis.db.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ashlar', '0013_auto_20150514_1712'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='record',
+            options={'ordering': ('-created',)},
+        ),
+        migrations.AlterField(
```

Comment:
The srid was changed to 4326 in #45. In the PR, it looks like a previous migration was changed to reflect this update rather than a new migration added. I wonder if your DB was still using the old srid, because you ran migrations with the old version of that migration before it was updated. I suspect if you destroyed/recreated your DB VM and then ran makemigrations, these calls to AlterField wouldn't be generated.